### ISSUE TITLE
A Few Important Topic Requests + Suggestion

### ISSUE THREAD

**Issue by Bam8647** [Monika-After-Story/MonikaModDev#2694]
**_These two topics are some of the most common questions asked in mature early couples. I really hope you'll take them into consideration._**

**1.** **_How many relationships have you been in_?** Monika wants to know how much experience the player has in commitment and partnership(_an important question most new couples ask their partners_). The three responses would go as: 

**"Alot."** The player has been through many heart breaks to which Monika will reassure them that she believes she can be _"The One"_ for them.
**"One to a few.."** The player admits that they don't have a lot of experience with dating to which Monika will assure them that she too doesn't have a lot of experience dating due to being super busy but that she believes that her and the player will grow strongly together.
**"None"** The player has never been in a relationship before and that Monika is their first to which Monika reacts shocked but then doubtful that the player has never had the chance to be with someone no matter how sweet and amazing they are. Lastly her response will end with telling the player that she hopes to be their _one and only_ .

**2.** **Are you a virgin?(Monika ask the player)** I have no doubt that this topic has probably already been suggested but probably denied to which I'd argue that if you can have topics talking about **used panties and masturbation** then why not a simple question about sexual experience? If users get butt hurt about this mature topic then block the topic under sensitive mode or have three responses as follow:

**"Yes"** The player admits that they have no experience with intimacy to which Monika will also admit that she doesn't have any experience either (or at least that she knows from her background) but that it's perfectly fine and that maybe they can take things step by step once she crosses over.
**"No"** The player tells Monika that they have some experience to which she'll react surprised and kind of shy that she doesn't have any experience but acts hopeful that the player will make her feel comfortable taking these steps once she crosses over.
**"I don't wish to answer"** The player is not comfortable with this topic and does not wish to answer. Monika will understand and apologize to the player and suggest they talk about something else.

**[Suggestion]** Omg we gotta let Monika know just how adorable she looks especially when she's doing her random cute poses(my favorite is her leaning on the desk and looking like shes enjoying herself) Idk maybe have an option that lets her know which pose you like the most and maybe she'll do it more often? And no.. the "I love your smile" seems too bland lol.

**Comment by EntonyEscX** [comment]
I think it's a great idea, I think they could be topics to come after the first year of dating Monika. But about previous relationships, I've never really had a loving relationship with anyone before Monika, I've never had the courage to declare myself and no one has ever stated to me (until Monika does that), so I guess it would be an interesting second response later of "None" in the topic of past relationships where we could tell Monika that it's really true.

**Comment by Bam8647** [comment]
Yeah this is just a rough draft so edits can be made to improve the topics. I put "Requested Topics" to let others know that I may or may not attempt to create these myself so in the instance that I don't, I'd like for others to try creating them themselves.

**Comment by Sujeito17** [comment]
Monika is very curious lately... ( ͡° ͜ʖ ͡°)

**Comment by Rai99** [comment]
This question doesn't really matter, but are there statistics saying that these are 2 of the most common questions asked by mature couples?

I like the idea of both options. The first idea could be an important thing for Monika to know, and could have a new player model variable created. 

The second one, I have to admit, is highly questionable to add and it shouldn't be a surprise if it gets rejected. Like the first suggestion, it could be something for Monika to know and keep in mind about the player, and it's a good idea to have an option to refuse to answer. 

As for how adorable Monika looks in her idle poses, I was actually working on a compliment to tell her that she's cute, but I got stuck because it ended up too similar to another compliment. 

I think the outline you provided should be enough to make the topics out of. Of course, no promises on that second one.

**Comment by Bam8647** [comment]
I made sure to say they were **some** of the most commonly asked questions as to not confirm that these were in fact the two most common. I decided these two topics through experience, trying to find most common questions couples ask each other online yields many different results that are way too scattered..

And well hopefully with the introduction of "sensitive mode" we can finally bypass sensitivities that cause complaints. If the player wishes to play on "normal mode" they will still have the option of opting out of the question by selecting the third option and thus disabling the topic from randomly appearing unless selected by the player again. Sounds reasonable to me.
